.github/workflows/create_release.yml

```
@@ -0,0 +1,37 @@
+# yamllint --format github .github/workflows/create_release.yml
+---
+name: create_release
+
+# We create a release version on a trigger tag, regardless of if the commit is documentation-only.
+#
+# See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+on:
+  push:
+    tags: 'release-[0-9]+.[0-9]+.[0-9]+**'  # Ex. release-1.2.3
+
+jobs:
+  create_release:
+    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:  # GH_TOKEN prevents the build of N.M.L tag, which we don't want to deploy anyway
```

@anuraaga @reta fyi I noticed that despite debugging and making this work earlier, it actually was better broke!

```suggestion
        with:  # GITHUB_TOKEN does not trigger when pushing tags, but we don't want conflict on the N.M.L tag anyway
```